blogs/converter.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const marked = require('marked');
+
+var listOfDirectoryFiles = fs.readdirSync('markdown-blogs');
+
+console.log(listOfDirectoryFiles);
+
+var listOfMarkdownFiles = [];
+//find the markdown files and put them in their own array
+for(let i = 0; i < listOfDirectoryFiles.length; i++){
+   const currentFileName = listOfDirectoryFiles[i];
+    const isMarkdown = currentFileName.indexOf(".md");
+    if(isMarkdown != -1){
+
+        listOfMarkdownFiles.push(currentFileName);
+    }
+}
+
+console.log(listOfMarkdownFiles);
+
+for(let i =0; i< listOfMarkdownFiles.length; i++){
+    const currentFileName = listOfMarkdownFiles[i];
+    //store the whole markdown file contents in currentFileBuffer
+   const currentFileBuffer = fs.readFileSync('markdown-blogs/' + currentFileName);
+    //Turn content to string
+    const fileContent = currentFileBuffer.toString();
```

Comment:
instead of `toString` here, when you read the file you can read as `utf8`
```
fs.readFileSync('markdown-blogs/' + currentFileName, 'utf8`);
```
Then see template literal comment too